src/graphql/search/datasets/normalize.ts

```diff
@@ -143,31 +143,33 @@ function getProperties(openApiData: any) {
   }
 }
 
-function getFacetOptions(facets: any, property: any) {
+function getFacetOptions(property: any, facets?: any) {
   const options: Array<CatalogFilterOptionsType> = getOptionsFromProperty(property)
 
   return options.map(({ label, id }) => ({
     id,
     label: label ? label : id,
-    count: Object.values(facets).reduce((acc, value: any) => value[id] || acc, 0) as number,
+    count: facets
+      ? (Object.values(facets).reduce((acc, value: any) => value[id] || acc, 0) as number)
```

Comment:
What happens here if `value[id]` is `0`? Do we want to fall back to `acc` in this case?